packages/testing/testing-module.builder.ts

```
@@ -27,31 +27,31 @@ export class TestingModuleBuilder {
     this.module = this.createModule(metadata);
   }
 
-  public overridePipe(typeOrToken): OverrideBy {
+  public overridePipe<T = any>(typeOrToken: T): OverrideBy {
     return this.override(typeOrToken, false);
   }
 
-  public overrideFilter(typeOrToken): OverrideBy {
+  public overrideFilter<T = any>(typeOrToken: T): OverrideBy {
     return this.override(typeOrToken, false);
   }
 
-  public overrideGuard(typeOrToken): OverrideBy {
+  public overrideGuard<T = any>(typeOrToken: T): OverrideBy {
     return this.override(typeOrToken, false);
   }
 
-  public overrideInterceptor(typeOrToken): OverrideBy {
+  public overrideInterceptor<T = any>(typeOrToken: T): OverrideBy {
     return this.override(typeOrToken, false);
   }
 
   /** @deprecated */
-  public overrideComponent(typeOrToken): OverrideBy {
+  public overrideComponent<T = any>(typeOrToken: T): OverrideBy {
```

Should this still exist? Shouldn't we just let it fail in case someone actually uses `components`?

---

I think that all changes that have been made so far, don't have to be considered as "breaking changes" which means that we wouldn't have to bump major version. That thing on the other hand (`overrideComponent`) should be probably removed in another PR :D

---

Oh I see.. Then I am fine with this PR :)